dfget/locator/static_locator.go

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright The Dragonfly Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package locator
+
+import (
+	"math/rand"
+	"sync/atomic"
+	"time"
+
+	"github.com/dragonflyoss/Dragonfly/dfget/config"
+	"github.com/dragonflyoss/Dragonfly/dfget/util"
+	"github.com/dragonflyoss/Dragonfly/pkg/netutils"
+)
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Maybe we can consider moving `rand.Seed(time.Now().UnixNano())` to the beginning of the main function to ensure that it always work and also avoids calling it everywhere. 😕 

```go
func main() {
	rand.Seed(time.Now().UnixNano())
	
	app.Execute()
}
```

Reply:
How about move `rand.Seed` to the top package of `dfget` rather than `cmd`? I think this operation should be transparent to the caller.

Reply:
Sure. That's OK with me. Just don't like calling `rand.Seed` everywhere!

Reply:
In order to ensure the independence of this pr, I open another pr #1318  to do this, please take a look.